hw5/bank.js

```diff
@@ -0,0 +1,94 @@
+let balance = 1000;
+
+alert(' Welcome to the Bank of An, Press W for Withdrawls, D for Deposits, B to see current Balance, or Q to Quit.')
+
+document.onkeydown = function (e) {
```

Comment:
I like the convention of using e for an event. Make sure that there isn't a space between the function and the params. function(e)

onkeydown might not be the best way of doing this, but it works...I'd use onkeyup instead, but that's more of a convention and opinion than anything.